### ISSUE TITLE
Shared Element Stack nested in MaterialTopTabNavigator not working

### ISSUE THREAD

**Issue by kyle-belle** [IjzerenHein/react-navigation-shared-element#170]
When i have my shared element Stack Nested in a MaterialTopTabNavigator the transition does not work. instead it just looks like a normal navigation

Exact same code somewhat working when the Share Element Stack Nav is return instead of from the Tab navigator

Not Working:
https://user-images.githubusercontent.com/43228761/120902147-c3b10e80-c60c-11eb-9519-b6caf5a1a3ff.mp4

Somewhat Working:
https://user-images.githubusercontent.com/43228761/120902251-36ba8500-c60d-11eb-9a28-231043a1d230.mp4

the difference in navigation "tree":
Not Working:
1) Stack Nav
2) -- MaterialTopTabNavigator
3) -- -- Shared Element Stack Nav

Somewhat Working:
1) Stack Nav
2) -- Shared Element Stack Nav

in fact in the not working example if i just replace "createMaterialTopTabNavigator" with "createStackNavigator" the Not Working example works so now:
Somewhat Working:
1) Stack Nav
2) -- Stack Nav
3) -- -- Shared Element Stack Nav

**Comment by kinhbang89** [comment]
oh yes I am experiencing the same issue now :P. Have you got any solutions for this problem?

**Comment by kyle-belle** [comment]
> oh yes I am experiencing the same issue now :P. Have you got any solutions for this problem?

Nah. Honestly if you really want it you may have to create your own tabbed system using horizontal flatlists which is a little extreme. 

I really don't think this will be fixed anytime soon based on the attention it got so far

**Comment by kinhbang89** [comment]
> > oh yes I am experiencing the same issue now :P. Have you got any solutions for this problem?
> 
> Nah. Honestly if you really want it you may have to create your own tabbed system using horizontal flatlists which is a little extreme.
> 
> I really don't think this will be fixed anytime soon based on the attention it got so far

Yes I am just  using now the https://github.com/satya164/react-native-tab-view and everything seems to work pretty well

**Comment by mixmaker** [comment]
I have a similar issue, I'm using bottom-tabs navigator nested inside a stack navigator. When I replace the parent createStackNavigator with createSharedElementStackNavigator, the ui becomes extremely glitchy, I cannot scroll on my app, cannot access my other stack screens. Also seems like there isn't many articles addressing this issue.
